### ISSUE TITLE
Maximum recursion depth exceeded

### ISSUE THREAD

**Issue by Nirajn2311** [thehappydinoa/iOSRestrictionBruteForce#4]
When i run the pyhton file, inget a runtime error maximum recursion depth exceeded. Could you please help me with the problem

**Comment by thehappydinoa** [comment]
Please paste in error bellow

**Comment by Nirajn2311** [comment]
The error:

```
python ioscrack.py -a

 iOSRestrictionBruteForce
 Written by thehappydinoa


Looking for backups in C:\Users\Nandish\AppData\Roaming\Apple Computer\MobileSync\Backup\/... Directory Found
Found 9736e0d3a918a53bac6d16f1f647c435b44a41f5
Found 999de05a64dbb9c3a5b585b4965a3ac52ed1e2a8
Traceback (most recent call last):
  File "ioscrack.py", line 32, in <module>
    main()
  File "ioscrack.py", line 18, in main
    crackHashes(findHashes(BACKUP_PATHS))
  File "C:\Users\Nandish\Desktop\VSCode\Python\iOSRestrictionBruteForce\ioscrack\parse.py", line 17, in findHashes
    device = iDevice(path + bkup_dir)
  File "C:\Users\Nandish\Desktop\VSCode\Python\iOSRestrictionBruteForce\ioscrack\idevice.py", line 22, in __init__
    self.findSecretKeySalt()
  File "C:\Users\Nandish\Desktop\VSCode\Python\iOSRestrictionBruteForce\ioscrack\idevice.py", line 34, in findSecretKeySalt
    return self.findSecretKeySalt(newPath=(not newPath))
  File "C:\Users\Nandish\Desktop\VSCode\Python\iOSRestrictionBruteForce\ioscrack\idevice.py", line 34, in findSecretKeySalt
    return self.findSecretKeySalt(newPath=(not newPath))
  File "C:\Users\Nandish\Desktop\VSCode\Python\iOSRestrictionBruteForce\ioscrack\idevice.py", line 34, in findSecretKeySalt
    return self.findSecretKeySalt(newPath=(not newPath))
 File "C:\Users\Nandish\Desktop\VSCode\Python\iOSRestrictionBruteForce\ioscrack\idevice.py", line 32, in findSecretKeySalt
    passfile = open(self.path + restrictionsFile, "r")
RuntimeError: maximum recursion depth exceeded
```

The 'line 34' repeats around a 1000 times and then it ends with the above error.

**Comment by thehappydinoa** [comment]
Fixed in 70b6e713ee5880fa1cb06af56ce109b1de5b9274

**Comment by Nirajn2311** [comment]
Thx 👍 👍
